ha/monitor/k8s/object_monitor.py

```diff
@@ -187,36 +187,41 @@ def _is_published_alert(self, alert) -> bool:
             True if it is published already
             False if it is a new alert
         """
-        if not isinstance(alert, dict):
-            return False
+        if isinstance(alert, str):
+            alert = json.loads(alert)
 
         incoming_alert = copy.deepcopy(alert)
 
-        header = incoming_alert["event"]["header"]
-        payload = incoming_alert["event"]["payload"]
```

Comment:
@mukhtar-inamdar 'event' is not present now in alert.

Reply:
ok